src/registrar/views/domain.py

```diff
@@ -837,6 +837,23 @@ def _send_domain_invitation_email(self, email: str, requestor: User, add_success
             )
             return None
 
+        # Check to see if an invite has already been sent
+        try:
+            invite = DomainInvitation.objects.get(email=email, domain=self.object)
+            # check if the invite has already been accepted
+            if invite.status == DomainInvitation.DomainInvitationStatus.RETRIEVED:
+                add_success = False
+                messages.warning(
+                    self.request,
+                    f"{email} is already a manager for this domain.",
```

Comment:
@Katherine-Osos want to review the content on this error message? Or is it mild enough you want to skip it?

Reply:
This message looks good!